.travis.yml

```diff
@@ -16,4 +16,4 @@ cache:
         - node_modules
 
 script:
-    - npm test
+    - npm run lint
```

Comment:
Use `npm test` so that the lint, clean and build all happen. When we add some unit testing, we can just update the package.json